common/po/fr.po

```diff
@@ -9,8 +10,8 @@ msgstr ""
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"PO-Revision-Date: 2018-01-25 08:40+0000\n"
-"Last-Translator: khowell <khowell@redhat.com>\n"
+"PO-Revision-Date: 2018-02-27 04:43+0000\n"
+"Last-Translator: Copied by Zanata <copied-by-zanata@zanata.org>\n"
```

Comment:
Is this something we want to avoid somehow?

Reply:
No biggie IMO.